backend/src/User.ts

```diff
@@ -5,9 +5,60 @@ import debug from 'debug'
 import Logger from './Logger'
 import EventBus from './EventBus'
 import path from 'path'
-import { r3 } from './remote.it'
+import axios from 'axios'
+import { API_URL, DEVELOPER_KEY } from './sharedCopy/constants'
+
+const defaults = {
+  apiURL: 'https://api.remot3.it/apv/v27',
+  successURL: 'https://app.remote.it', // 'https://' + window.location.host
+  taskQueue: 'WeavedTaskQueue',
+  jobQueue: 'WeavedJobQueue',
+}
+
+const instance = setupAxios({ apiURL: API_URL, developerKey: DEVELOPER_KEY })
+instance.interceptors.request.use(
```

Comment:
Please see what code is totally needed, and remove anything that is not required.